<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatDialog</class>
 <widget class="QDialog" name="GroupChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg/groupChatDialogBg.png);</string>
   </property>
   <widget class="QTextEdit" name="SendMessageText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="FriendListWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="CloseWinBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/closeBtn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="MinWinBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/bg/minBtn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMessageBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="messageHistoryBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>70</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/chatHistory.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shakeBtn">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>410</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/shake.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fontBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/font.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="faceBtn">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>410</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/face.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cutPhoto">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>410</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/cutPhoto.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="senImageBtn">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>410</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/sendImage.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="blankBtn">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>410</y>
      <width>168</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/chatBlank.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scrawlBtn">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>410</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg/content.png);
border-image: url(:/btn/scrawl.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWebView" name="messageListWebView">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>89</y>
      <width>471</width>
      <height>271</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MinWinBtn</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
